Tasks/CopyFilesV2/Tests/L0.ts

```diff
@@ -199,6 +199,7 @@ describe('CopyFiles L0 Suite', function () {
         assert(
             runner.succeeded,
             'should have succeeded');
+        assert(runner.stdOutContained(`Calling fs.utimes on ${path.normalize('/destDir')}`));
```

Comment:
Nit - add a second parameter to provide a message if this fails (e.g. `'should have copied timestamp'`)